interchain-exchange/tutorial/09-walkthrough.md

```diff
@@ -18,8 +18,8 @@ Start the networks, as input use the `mars.yml` and `venus.yml` for each blockch
 
 ```bash
 # start networks
-starport serve -c mars.yml -r
-starport serve -c venus.yml -r
+starport serve -c interchain/mars.yml -r
```

Comment:
```suggestion
starport serve -c interchange/mars.yml -r
```

Reply:
got it, then there is a typo in the intro where the `interchain`  folder was mentioned, let me revert the changes and update the pr